tests/storage/storage.go

```diff
@@ -315,8 +318,10 @@ var _ = SIGDescribe("Storage", func() {
 					libwait.WaitForVirtualMachineToDisappearWithTimeout(vmi, 120)
 				}
 			},
-				Entry("[test_id:3132]with Disk PVC", newRandomVMIWithPVC),
-				Entry("[test_id:3133]with CDRom PVC", newRandomVMIWithCDRom),
+				Entry("[test_id:3132]with Disk PVC", newRandomVMIWithPVC, nil),
+				Entry("[test_id:3133]with CDRom PVC", newRandomVMIWithCDRom, nil),
+				Entry("with Disk PVC and terminationGracePeriod = 0", newRandomVMIWithPVC, pointer.P(int64(0))),
+				Entry("with CDRom PVC and terminationGracePeriod = 0", newRandomVMIWithCDRom, pointer.P(int64(0))),
```

Comment:
Why do we need these tests? Are we really worried about this scenario?
Anyway, note this will flake often as it does today